nixos/modules/services/continuous-integration/gitlab-runner/runner.md

```diff
@@ -0,0 +1,72 @@
+# GitLab Runner {#module-services-gitlab-runner}
+
+GitLab Runner is a CI runner which is an executable which you can host yourself.
+A Gitlab pipeline runs operations over a Gitlab Runner. These can include
+building an executable, running a test suite, pushing a docker image, etc. The
+Gitlab Runner receives jobs from Gitlab which it then dispatches to the
+configured executors
+([`docker` (`podman`), or `shell` or `kubernetes`](https://docs.gitlab.com/runner/executors)).
+
+The
+[services.gitlab-runner.services](https://search.nixos.org/options?query=services.gitlab-runner.services)
+documents a number of typical setups to configure multiple runners with
+different executors. The [below example](#ex-gitlab-runner-podman) gives a **more elaborate** example how to
+configure a Gitlab Runner with caching and reasonably good security practices.
+
+::: {#ex-gitlab-runner-podman .example}
+
+## Example Gitlab Runner with `podman` and Nix Store Caching
+
+The [example](./runner.nix) configures a Gitlab Runner with the following
+features:
+
+- The executor is `podman` which gives you better additional safety than
+  `docker`. That means every job is run in a `podman` container.
+
+- The following container **images** are built with Nix:
+
+  **For Gitlab Jobs**:
+
+  - an image based on Ubuntu (name: `local/alpine`) with a Nix installation used
```

Comment:
There is a mixup with the names here.
You call the Ubuntu image `local/alpine` and vise versa.